apps/wpcom-block-editor/package.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "@automattic/wpcom-block-editor",
+	"version": "1.0.0-alpha.0",
+	"description": "Utilities for the block editor in WordPress.com integration",
+	"sideEffects": true,
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/Automattic/wp-calypso.git",
+		"directory": "apps/wpcom-block-editor"
+	},
+	"private": true,
+	"author": "Automattic, Inc.",
+	"license": "GPL-2.0-or-later",
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"scripts": {
+		"clean": "npx rimraf dist",
+		"prebuild": "npm run clean",
+		"build": "webpack --config='../../packages/calypso-build/webpack.config.js' --output-path='./dist' --env.WP='true' iframe-bridge-server='./src/iframe-bridge-server.js' "
```

Comment:
BTW if in future you want to just build to external folder, you could have:
```
"bundle": "webpack --config='../../packages/calypso-build/webpack.config.js' --env.WP='true' iframe-bridge-server='./src/iframe-bridge-server.js'",
"build": "npm run bundle -- --output-path=./dist",
"build-sandbox": "npm run bundle"
```

and then be able to pass [Webpack CLI](https://webpack.js.org/api/cli/) args to the builder:

```
npx lerna run build-sandbox --scope='@automattic/wpcom-block-editor' -- -- --output-path=/path_to_sandbox
```

(Wonky double `--` is needed for first skipping Lerna args and then NPM args to reach Webpack...)

For some reason `--watch` didn't work here as I expected.

Nit; extra space at the end of your script. :-)

Reply:
Building to an external folder that we pass in as an arg would also allow us to build to a local copy of our sandbox – which in turn could sync automatically with Unison (rsync without needing the call to rsync 😜).

Reply:
Nice suggestion @simison! I applied it in d5108de given that we will actually need that since my plan is to build this package with CircleCI and store it in an artifact (like we do with Notifications).